src/utils/TypeGuards.ts

```diff
@@ -412,6 +412,44 @@ export class TypeGuards {
         return node.getKind() === SyntaxKind.DebuggerStatement;
     }
 
+    /**
+     * Gets if the node is a declaration.
+     * @param node - Node to check.
+     */
+    static isDeclaration(node: compiler.Node): node is compiler.ArgumentedNode & compiler.Node {

```

Comment:
See the warning at the top of this file. The `is` methods of this file are code generated so this will be wiped out next time the code generation runs.